spec/models/room_spec.rb

```diff
@@ -50,6 +50,15 @@
     end
   end
 
+  describe '#primary_jam' do
+    it 'returns the most recently promoted jam' do
+      create(:jam, room: room, promoted_at: 1.minute.ago)
+      primary_jam = create(:jam, room: room, promoted_at: Time.current)
```

Comment:
rule of thumb, you want to bracket it to make sure it just doesn't get the latest created. so I'd add one more after this `create(:jam, room: room, promoted_at: 1.minute.ago)`